transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannel.java

```diff
@@ -53,17 +52,11 @@ public EpollServerSocketChannel(int fd) {
 
     EpollServerSocketChannel(LinuxSocket fd) {
         super(fd);
-        // As we create an EpollServerSocketChannel from a FileDescriptor we should try to obtain the remote and local
-        // address from it. This is needed as the FileDescriptor may be bound already.
-        local = fd.localAddress();
```

Comment:
This is the assumption I always had I think we not account for have it "asynchronous" in the first place.